Source/Tools/LinuxEmulation/LinuxSyscalls/x32/FD.cpp

```diff
@@ -275,7 +280,12 @@ namespace FEX::HLE::x32 {
         sigsetsize);
 
       if (timeout_ts) {
-        *timeout_ts = tp64;
+        struct timespec32 timeout{};
+        timeout = tp64;
+
+        // We read the timeout correct, but now we can't write it. This can happen when then timeout is in a read-only memory segment.
```

Comment:
This comment explains exceptional behavior as if it were the common case. Normally, you *can* write the timeout.

Reply:
moved comment.